packages/sandbox/e2e/cli/configuration.test.ts

```diff
@@ -28,7 +28,7 @@ describe('the CLI configuration in default mode', () => {
 
   it('should have sandbox preset of global options in a fresh environment', async () => {
     const { exitCode, stdout, stderr } = await emptyPrjSandbox.$init();
-    const { collect, persist, assert } = getInitCommandOptionsFromArgv(SANDBOX_PRESET);
+    const { collect, persist, assert } = getInitCommandOptionsFromArgv(SANDBOX_PRESET as any);
```

Comment:
As i Removed the incorrect types and the test are built on the incorrect types I have done this. 

I plan to fix it before merging. 

The same goes for the rest of the type changes in the end to end test!